fastn-core/src/auth/github/mod.rs

```diff
@@ -73,44 +88,129 @@ pub async fn callback(
     // ask to update details by giving a form
     // redirect to next for now
     if fastn_core::auth::utils::is_authenticated(req) {
+        // [SUCCESS] logging: already authenticated
+        let log_success_message = "user: already authenticated".to_string();
```

Comment:
Do log this kind of text, use text like `user-already-authenticated`.

Reply:
Are you not setting the outcome? Outcome must be "user-error" in this case, and if I am interested in precisely what user error then I will want user-already-authenticated string. Normally I want to see how many user error happened today, vs how many system error.

Reply:
Read the comments on Event class definition in models.py file.